packages/faceted-search/src/components/BasicSearch/BasicSearch.component.js

```diff
@@ -60,34 +61,41 @@ const BasicSearch = ({
 	const badgeFacetedContextValue = { state, dispatch, onSubmit };
 	return (
 		<div id={basicSearchId} className={css('tc-basic-search')}>
-			<BadgeFacetedProvider value={badgeFacetedContextValue}>
-				<BadgesGenerator
-					badges={state.badges}
-					badgesDictionary={badgesDictionary}
-					getBadgeFromDict={getBadgesFromDict}
-					id={basicSearchId}
-					t={t}
-				/>
-			</BadgeFacetedProvider>
-			<BadgeOverlay
-				id={basicSearchId}
-				iconName="plus-circle"
-				label={t('OPEN_ADD_FACET_BUTTON', { defaultValue: 'Add filter' })}
-				t={t}
-				hideLabel
-				hasAddButton
-			>
-				{setOverlayOpened => (
-					<AddFacetPopover
+			<div className={css('tc-basic-search-content')}>
```

Comment:
Why does this new div has been added ? Is it possible to only have 1 search div, with the badge generator + add button + clear button in the row flow ?

Reply:
I add a link to the JIRA with a screenshot on it, think it could help (clear button is on the right)

Reply:
The border is more linked to the BasicSearch div container.
IMO you can remove this button wrapping div, and add some css on the `.tc-basic-search`
```css
.tc-basic-search:after {
    content: '';
    height: 2.5rem;
    width: 0;
    border: 1px solid $gallery; /* keep px here, we don't want to increase the border with html size increase */

    /* placement: center vertically */
    position: absolute; /* the div must have position != static */
    top: 50%;
    transform: translateY(-50%);
}
```

Reply:
Good point, PR updated. I just put this on `tc-basic-search-clear-button:after`: `tc-basic-search` end before the button, it put the border on left on it (we want on right)